src/opencl_krb5pa-sha1_fmt_plug.c

```diff
@@ -240,7 +240,7 @@ static void done(void)
 
 static void init(struct fmt_main *_self)
 {
-	static char valgo[sizeof(ALGORITHM_NAME) + 8] = "";
+	static char valgo[sizeof(ALGORITHM_NAME) + 12] = "";
```

Comment:
I hate these fixes. We know that %u can't be larger than 16 (so `+ 4` is actually enough) but the ridiculous compiler false-positive-warning-engine[TM] complains because it thinks %u can be  ten digits. But yes, this is the easiest fix of course.

Reply:
I don't like these fixes myself but we have to live with GCC :(

These changes are still not enough and I will have more coming in a new PR.

Also, I am not sure if my `strncpy` fixes in this PR are 100% correct. At least, they pass CI testing.